py3status/modules/getjson.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+Query a url and display the json response
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds
+        (default 30)
+    format: placeholders will be replaced by the returned json key names
+        (default '')
+    negative_cache_timeout: how often to check again when offline
+        (default 2)
+    params: specify the parameters to pass to the url as a dict
+        (default {})
```

Comment:
why not just add these in the url?  If they were POST I could understand but these are GET